.github/workflows/newfold-prepare-release.yml

```diff
@@ -0,0 +1,28 @@
+name: Newfold Prepare Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      level:
+        description: The level of release to be used.
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+        default: "patch"
+        required: true
+
```

Comment:
A default permission set of `permissions: {}` is missing for this one.

I believe that we may need to set `permissions` for for `pull-requests` and `contents` to `write` in order for the called workflow to have the necessary permissions. Called workflows cannot escalate their permissions past what they inherit from the calling workflow.